terraform/github-to-gitlab-mirror/update-repo.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+# This script updates a gitlab repository mirror from github
+if [ -z "$1" ]; then
+    echo "Usage: $0 <repo-name>"
+    exit 1
+fi
+
+REPO_NAME=$1
+tempdir=$(mktemp -d)
+trap "rm -rf '$tempdir'" EXIT
+cd "$tempdir"
+
+git clone --mirror "https://github.com/$REPO_NAME" .
+git push --force --mirror "https://gitlab.com/$REPO_NAME"
```

Comment:
Not yet quite sure how this can be done in CI.
We would probably need a bot account.

Reply:
Could use a hercules effect on a schedule?

Reply:
Ah nice. Haven't seen this feature: https://blog.hercules-ci.com/2022/11/15/easy-updates-with-onschedule-0.9.8/